### ISSUE TITLE
Yellowish brown character to kidney stone suggestive of cystine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11442]
**Preferred term label:**
Yellowish brown character to kidney stone suggestive of cystine

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/books/NBK619248/ Yellowish brown character to kidney stone suggestive of cystine, often has a granular surface

This finding is important because it allows a rapid identification of the type of nephrolithiasis.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cystinuria

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal  This term already existed
[Cystine nephrolithiasis HP:6001031](https://hpo.jax.org/browse/term/HP:6001031)
I have updated the definition:
A cystine stone is a type of kidney stone that is yellowish brown with a granular surface. They are observed in individuals with Cystinuria and tend to be of early onset, to occur repeatedly, and to be associated with urinary tract infections.

I think it is possible that your term request refers to the very first phenotypic observation. I have added the "Yellowish brown character to kidney stone suggestive of cystine" as a RELATED_SYNONYM to at least partially cover this. If you require more detail, please reopen the issue.
